sphinx_rtd_theme/layout.html

```diff
@@ -105,136 +105,6 @@
 </head>
 
 <body class="wy-body-for-nav">
```

Comment:
I think this should be moved to the body include and inside a `{% block body %}`

Reply:
A block with name body exists already.